add relative path snippet (#6547)
fishtown-analytics/docs.getdbt.com: website/docs/reference/project-configs/seed-paths.md

## Default

> By default, dbt expects seeds to be located in the `seeds` directory, i.e. `seed-paths: ["seeds"]`. 

Reviewer: ```suggestion
By default, dbt expects seeds to be located in the `seeds` directory. For example, `seed-paths: ["seeds"]`. 
```
